molecule/default/tests/test_default.py

```diff
@@ -21,3 +21,12 @@ def test_admin_authorized_keys_file(host):
     assert f.group == 'admin'
     assert f.contains('ADMIN_USER_1')
     assert f.contains('ADMIN_USER_2')
+
+
+def test_bash_history_config(host):
```

Comment:
@speedlight :+1:

Reply:
I notice that currently could be difficult to test using role defined variables or facts. An issue like these is here: https://github.com/philpep/testinfra/issues/345

In different places people is using Testaid: https://github.com/RebelCodeBase/testaid

I think we could use it as well so in this case we can `assert f.contains('HISTSIZE=6000')` only when `sb_debian_base_bash_history` is set to true.

What do you think? 

UPDATE:
I think the problem with testaid for us is this:
https://github.com/RebelCodeBase/testaid/blob/0c9af2a068b743449d7d93aeb3b581a8b7b7c6b8/setup.py#L22